### ISSUE TITLE
Invalid AST

### ISSUE THREAD

**Issue by vdka** [kai-language/kai#17]
```
_, y.arr[0] = 5, 8
```
Creates the tree:
```
(memberAccess
      (multiple
        (dispose)
        (identifier(y)))
      (assignment(=)
        (subscript
          (identifier(arr))
          (integer(0)))
        (multiple
          (integer(5))
          (integer(8)))))
```

The correct tree, I think is:
```
(assignment(=)
  (multiple
    (dispose)
    (subscript
      (memberAccess
        (identifier(y))
        (identifier(arr)))
      (integer(0))))
  (multiple
    (integer(5))
    (integer(8))))
```

**Comment by vdka** [comment]
Even worse:
`x, y := 5, 6` no longer parses too
```
    (multipleDeclaration
      (multiple
        (declaration((x: ??)))
        (declaration((y: ??))))
      (multiple
        (integer(5))
        (integer(6))))
    (assignment(=)
      (multiple
        (dispose)
        (identifier(y)))
      (multiple
        (identifier(y))
        (identifier(x))))
```
Instead we get:
```
    (multiple
      (identifier(x))
      (declaration((y: ??))
        (multiple
          (integer(5))
          (integer(6))))))
```

**Comment by vdka** [comment]
Closed by 258ee50
